### PR TITLE
Change deploy tag to circleci build number

### PR DESCRIPTION
**Pull request by davish**
Hopefully now the rebuild webhook will properly redeploy the site with Ghost changes (and soon, with Platform changes too).

Not sure if I did this right, but any feedback will be appreciated 😄

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -27,4 +27,5 @@ workflows:
           requires:
             - docker/publish
           context: k8s-deploy
+          deploy-tag: '${CIRCLE_BUILD_NUM}'
```
Comment (pawalt) [inline .circleci/config.yml]: use double quotes

**Review by pawalt — CHANGES_REQUESTED** [review]
You also need to change the docker image to be tagged using circle build num

File: .circleci/config.yml [hunk 2]
```diff
@@ -19,12 +19,13 @@ workflows:
           requires:
             - react-tools/check
           image: pennlabs/website
-          tag: '${CIRCLE_SHA1},latest'
+          tag: '${CIRCLE_SHA1},${CIRCLE_BUILD_NUM},latest'
```
Comment (ArmaanT) [inline .circleci/config.yml]: We probably don't need to tag on SHA any more

Comment (pawalt) [reply .circleci/config.yml]: Agreed. SHA isn't meanginful since it doesn't capture everything about the image.

**Comment by pawalt** [comment]
LGTM

**Pull request merged**